src/test/java/com/wikia/webdriver/testcases/mercurytests/HTMLTitleTests.java

```diff
@@ -1,21 +1,114 @@
 package com.wikia.webdriver.testcases.mercurytests;
 
-
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.configuration.Configuration;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.Emulator;
 import com.wikia.webdriver.common.core.url.Page;
-import com.wikia.webdriver.common.dataprovider.MercurySEODataProvider;
-import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
 import com.wikia.webdriver.elements.common.Navigate;
 import com.wikia.webdriver.elements.mercury.components.Head;
+
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
 @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
-public class HTMLTitleTests extends TemplateNoFirstLoad {
+public class HTMLTitleTests extends NewTestTemplate {
 
+  /**
+   * [0] wikiName
+   * [1] path
+   * [2] expected title
+   */
+  private String[][] testCases = {
+    {
+      "sktest123",
+      "/wiki/Sktest123_Wiki",
+      "Sktest123 Wiki - Wikia"
+    },
+    {
+      "sktest123",
+      "/wiki/Style-5H2",
+      "Style-5H2 - Sktest123 Wiki - Wikia"
+    },
+    {
+      "sktest123",
+      "/wiki/TestDisplayTitle",
+      "testing abc - Sktest123 Wiki - Wikia"
+    },
+    {
+      "sktest123",
+      "/wiki/Category:Premium_Videos",
+      "Category:Premium Videos - Sktest123 Wiki - Wikia"
+    },
+    {
+      "sktest123",
+      "/wiki/Category:Non-premium_Videos",
+      "Category:Non-premium Videos - Sktest123 Wiki - Wikia"
+    },
+    {
+      "sktest123",
+      "/wiki/Category:Premium",
+      "PremiumVideos - Sktest123 Wiki - Wikia"
+    },
+    {
+      "es.pokemon",
+      "/wiki/WikiDex",
+      "WikiDex - Wikia"
+    },
+    {
+      "es.pokemon",
+      "/wiki/Lista_de_Pokémon",
+      "Lista de Pokémon - WikiDex - Wikia"
+    },
+    {
+      "es.pokemon",
+      "/wiki/Categoría:Regiones",
+      "Categoría:Regiones - WikiDex - Wikia"
+    },
+    {
+      "starwars",
+      "/wiki/Main_Page",
+      "Wookieepedia - Wikia"
+    },
+    {
+      "starwars",
+      "/wiki/Droid_starfighter",
+      "Droid starfighter - Wookieepedia - Wikia"
+    },
+    {
+      "starwars",
+      "/main/category/Future_films",
+      "Future_films"
+    },
+    {
+      "starwars",
+      "/main/section/Films",
+      "Films"
+    },
+    {
+      "mediawiki119",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "mediawiki119" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)